stdlib/public/SwiftShims/CMakeLists.txt

```diff
@@ -110,10 +96,10 @@ swift_install_in_component(clang-builtin-headers
     DESTINATION "lib/swift/clang"
     PATTERN "*.h")
 
-# Alternatively, install a symbolic link to the Clang resource directory.
-swift_install_in_component(clang-resource-dir-symlink
-    DIRECTORY "${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR}/lib/swift/install-tmp/install-tmp/clang"
-    DESTINATION "lib/swift")
+swift_install_symlink_component(clang-resource-dir-symlink
+  LINK_NAME clang
+  TARGET ../clang/${CLANG_VERSION}
```

Comment:
No. The target is the path the symlink points to, which is intentionally relative to where it gets installed to. That makes it so the install directory can be moved around or relocated without breaking.